src/main/java/org/jabref/gui/openoffice/ManageCitationsDialogView.java

```diff
@@ -0,0 +1,93 @@
+package org.jabref.gui.openoffice;
+
+import javafx.fxml.FXML;
+import javafx.scene.Node;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableColumn.CellEditEvent;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.TextFieldTableCell;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.text.Text;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.gui.util.ValueTableCellFactory;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.strings.StringUtil;
+
+import com.airhacks.afterburner.views.ViewLoader;
+import com.sun.star.beans.UnknownPropertyException;
+import com.sun.star.container.NoSuchElementException;
+import com.sun.star.lang.WrappedTargetException;
+
+public class ManageCitationsDialogView extends BaseDialog<Void> {
+
+    private static final String HTML_BOLD_END_TAG = "</b>";
+    private static final String HTML_BOLD_START_TAG = "<b>";
+
+    private final DialogService dialogService;
```

Comment:
You should be able to use `@Inject DialogService .. ̀  instead of constructor injection.